vendor/opam/src/core/opamCompat.pp.ml

```diff
@@ -0,0 +1,70 @@
+(**************************************************************************)
+(*                                                                        *)
+(*    Copyright 2018-2020 OCamlPro                                        *)
+(*                                                                        *)
+(*  All rights reserved. This file is distributed under the terms of the  *)
+(*  GNU Lesser General Public License version 2.1, with the special       *)
+(*  exception on linking described in the file LICENSE.                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+module String =
+#if OCAML_VERSION >= (4, 13, 0)
+  String
+#else
```

Comment:
There's a way to do this without cppo:

```
module String = struct
  include struct
    [@@@ocaml.warning "-32"]

    let exists = (* put your fallback implementation here *)
  end

  include String
end
```

Reply:
Good point, being worked on on the OPAM side, I will update once that is merged: https://github.com/ocaml/opam/pull/5508

Reply:
Rebased the `vendored` branch and updated the code here to use that commit.